src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -421,6 +424,25 @@ public final class Guardrails implements GuardrailsMBean
                      format("The keyspace %s has a replication factor of %s, above the %s threshold of %s.",
                             what, value, isWarning ? "warning" : "failure", threshold));
 
+    public static final MaxThreshold maximumAllowableTimestamp =
+    new MaxThreshold("timestamp_upper_bound",
```

Comment:
It seems this validation is still producing logging messages of the form `timestamp_upper_bound violated:`. Probably we want to use the same name that is used for the yaml property:
```suggestion
    new MaxThreshold("maximum_timestamp",
```
And same for the minimum timestamp.

Reply:
oversight will fix